src/core/Routes.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react'
 import { Switch, Route } from 'react-router-dom'
 
+import { Accordion } from 'common/components/Accordion/index.js'
+import { SearchResults } from 'common/components/SearchResults/index.js'
+
 import { HelloWorldClass } from './HelloWorld'
 import { HelloWorldFunctional } from './HelloWorld'
+import { PageTemplate } from './PageTemplate'
 
 // High-level routes for the application
 // Sub-routes should be nested within components
 export const Routes = () => (
   <Switch>
     <Route exact path='/'>
       <HelloWorldClass cheeseburger="cheeseburger" />
-      <HelloWorldFunctional hotdog="hotdog bun" isEnabled={false}>
+      <HelloWorldFunctional hotdog="hotdog bun" isEnabled>
         This is a child thing.
+        <Accordion title="Click click!"></Accordion>
```

Comment:
Don't forget to add `children` to the `Accordion`, otherwise you'll never see the content.

```<Accordion title="Click click!">content goes here</Accordion>```